lib/rucio/db/sqla/session.py

```diff
@@ -207,10 +211,10 @@ def get_engine():
             conv = mysql_convert_decimal_to_float(pymysql=sql_connection.startswith('mysql+pymysql'))
             params['connect_args'] = {'conv': conv}
         for param, param_type in config_params:
-            try:
-                params[param] = param_type(config_get(DATABASE_SECTION, param, check_config_table=False))
-            except:
-                pass
+            param_from_config = config_get(DATABASE_SECTION, param, check_config_table=False, raise_exception=False, default=None)
+            if param_from_config is not None:
+                params[param] = param_type(param_from_config)
```

Comment:
This should be in a separate commit (it's not a change that adds logging to a try/except block)